# Generated with xap_parser.py

# openPOWERLINK variables:
string shm_name
uint16 pid

# xap.h variables:
uint8 CN1_M00_X20PS9400a_NetworkStatus
bool CN1_M00_X20PS9400a_StatusInput01
bool CN1_M00_X20PS9400a_StatusInput02
uint8 CN1_M01_X20DI9371_NetworkStatus
uint16 CN1_M01_X20DI9371_DigitalInput
uint8 CN1_M02_X20DO9322_NetworkStatus
uint16 CN1_M02_X20DO9322_StatusInput01
